app/assets/stylesheets/master.scss

```diff
@@ -1 +1,19 @@
 @import "bootstrap";
+
```

Comment:
I'm going to go ahead and move all the styles into just master.scss, and we can save games.scss for other things, if we decide to use it at all.